### DISCUSSION TITLE
Newer version does not support my display. (S22C300)

### DISCUSSION THREAD

**Discussion by StormBlackwell** [xanderfrangos/twinkle-tray#680] [Monitor Compatibility Issues]
1.16.0 does not seem to work with my Samsung S22C300. 
16 shows no communication method
15.2 shows DDC/CI
As a side note, my HP M22f FHD was still working just fine. 

I Updated my graphics drivers, but that didn't fix it. 
I restarted Twinkle Tray and my pc, but that didn't fix it.
I reinstalled Twinkle Tray 1.16.0, but that didn't fix it.

I downgraded back to 1.15.2 and it works again. 
Hopefully the newer versions work for my older display again someday

**Comment by Ungaarr** [comment]
1.16 not work hotkeys Dell U2421E
